ui/src/app/components/help-sidebar.tsx

```diff
@@ -479,6 +499,11 @@ export const HelpSidebar = fp.flow(
       openZendeskWidget(givenName, familyName, username, contactEmail);
     }
 
+    navigateToTerminal() {
+      const {workspace: {id, namespace}} = this.props;
+      this.props.navigate(['workspaces', namespace, id, 'terminals']);
```

Comment:
The reason this is not preferred is because you can't get normal link behavior, i.e. ctrl-click to open in a new tab. If you really can't get the normal routelink working then, this will have to do for now (also, if this is an existing convention in the class)